2.0/README.md

```diff
@@ -0,0 +1,476 @@
+# Vector Tile Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+"SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in
+this document are to be interpreted as described in [RFC 2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+## 1. Purpose
+
+This document specifies a space-efficient encoding format for tiled geographic vector data. It is designed to be used in browsers or server-side applications for fast rendering or lookups of feature data.
+
+## 2. File Format
+
+The Vector Tile format uses [Google Protocol Buffers](https://developers.google.com/protocol-buffers/) as a encoding format. Protocol Buffers are a language-neutral, platform-neutral extensible mechanism for serializing structured data.
+
+## 2.1. File Extension
+
+The filename extension for Vector Tile files SHOULD be `mvt`. For example, a file might be named `vector.mvt`.
+
+## 2.2. Multipurpose Internet Mail Extensions (MIME)
+
+When serving Vector Tiles the MIME type SHOULD be `application/vnd.mapbox-vector-tile`.
```

Comment:
Agreed - I think it's better to have a defined, if unofficial, MIME type than wait for one the registration process to complete. The instruction is "SHOULD", so anyone who feels very strongly about using unapproved MIME types can always ignore it.